inc/classes/subscriber/class-google-tracking-cache-busting-subscriber.php

```diff
@@ -14,6 +14,7 @@
  * @author Remy Perona
  */
 class Google_Tracking_Cache_Busting_Subscriber implements Subscriber_Interface {
+
```

Comment:
We touched this file. Let's move and rename it per our new standards/architecture.

Reply:
Done in 4fc59ae.